x-pack/metricbeat/module/aws/_meta/kibana/7/dashboard/Metricbeat-aws-elb-overview.json

```
@@ -0,0 +1,995 @@
+              "w": 23,
+              "x": 0,
+              "y": 23
+            },
+            "panelIndex": "8",
+            "panelRefName": "panel_6",
+            "title": "Backend Connection Errors",
+            "version": "7.2.0"
+          },
+          {
+            "embeddableConfig": {},
+            "gridData": {
+              "h": 7,
+              "i": "9",
+              "w": 11,
+              "x": 0,
+              "y": 0
+            },
+            "panelIndex": "9",
+            "panelRefName": "panel_7",
+            "version": "7.2.0"
+          },
+          {
+            "embeddableConfig": {},
+            "gridData": {
+              "h": 9,
+              "i": "10",
+              "w": 25,
+              "x": 23,
+              "y": 23
+            },
+            "panelIndex": "10",
+            "panelRefName": "panel_8",
+            "title": "HTTP Backend 2XX",
+            "version": "7.2.0"
+          }
+        ],
+        "timeRestore": false,
+        "title": "[Metricbeat AWS] ELB Overview",
+        "version": 1
+      },
+      "id": "e74bf320-b3ce-11e9-87a4-078dbbae220d",
+      "migrationVersion": {
+        "dashboard": "7.0.0"
+      },
+      "references": [
+        {
+          "id": "b9703dd0-b3c9-11e9-87a4-078dbbae220d",
+          "name": "panel_0",
+          "type": "visualization"
+        },
+        {
+          "id": "d560de70-b3c7-11e9-87a4-078dbbae220d",
+          "name": "panel_1",
+          "type": "visualization"
+        },
+        {
+          "id": "6fc1efd0-b3c9-11e9-87a4-078dbbae220d",
+          "name": "panel_2",
+          "type": "visualization"
+        },
+        {
+          "id": "6392bc30-b3c9-11e9-87a4-078dbbae220d",
+          "name": "panel_3",
+          "type": "visualization"
+        },
+        {
+          "id": "b2ea15a0-b3c7-11e9-87a4-078dbbae220d",
+          "name": "panel_4",
+          "type": "visualization"
+        },
+        {
+          "id": "21f30090-b3ca-11e9-87a4-078dbbae220d",
+          "name": "panel_5",
+          "type": "visualization"
+        },
+        {
+          "id": "572d40e0-b3ca-11e9-87a4-078dbbae220d",
+          "name": "panel_6",
+          "type": "visualization"
+        },
+        {
+          "id": "b5308940-7347-11e9-816b-07687310a99a",
+          "name": "panel_7",
+          "type": "visualization"
+        },
+        {
+          "id": "1f528f50-b3ce-11e9-87a4-078dbbae220d",
+          "name": "panel_8",
+          "type": "visualization"
+        }
+      ],
+      "type": "dashboard",
+      "updated_at": "2019-08-01T15:09:06.696Z",
+      "version": "WzYzNSw4XQ=="
+    },
+    {
+      "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [],
+            "query": {
+              "language": "kuery",
+              "query": ""
+            }
+          }
+        },
+        "title": "AWS ELB HTTP 5XX Errors",
```

Visualization titles should have the metricset as suffix
```suggestion
        "title": "HTTP 5XX Errors [AWS ELB]",
```

---

👍 I will fix all visualizations. Thanks!

---

Oh sorry, I should have said `ELB HTTP 5XX Errors [Metricbeat AWS]`